src/test/java/ch/wisv/areafiftylan/controller/TeamRestControllerTest.java

```diff
@@ -0,0 +1,93 @@
+package ch.wisv.areafiftylan.controller;
+
+import ch.wisv.areafiftylan.Application;
+import ch.wisv.areafiftylan.model.User;
+import ch.wisv.areafiftylan.service.repository.TeamRepository;
+import ch.wisv.areafiftylan.service.repository.UserRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.TestRestTemplate;
+import org.springframework.boot.test.WebIntegrationTest;
+import org.springframework.http.*;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = Application.class)
+@WebIntegrationTest("server.port = 0")
+public class TeamRestControllerTest {
+
+    @Value("${local.server.port}")
+    int port;
+
+    public static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Autowired
+    private TeamRepository teamRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    private RestTemplate restTemplate = new TestRestTemplate();
+
+    private String serverPath;
+
+    @Before
+    public void init() {
+        teamRepository.deleteAll();
```

Comment:
Hier moeten we binnenkort HEEL voorzichtig mee gaan doen. Ik zou m vast weghalen, want de database leeghalen wil je eigenlijk nooit. Dit had ik er in gezet omdat we nu nog met een in-memory database werken.